drivers/bridge/bridge.go

```diff
@@ -205,29 +359,22 @@ func (d *driver) getNetwork(id types.UUID) (*bridgeNetwork, error) {
 	return d.network, nil
 }
 
-func parseNetworkOptions(option options.Generic) (*NetworkConfiguration, error) {
-	var config *NetworkConfiguration
-
-	genericData, ok := option[netlabel.GenericData]
-	if ok && genericData != nil {
-		switch opt := genericData.(type) {
-		case options.Generic:
-			opaqueConfig, err := options.GenerateFromModel(opt, &NetworkConfiguration{})
-			if err != nil {
+func parseNetworkOptions(option options.Generic) (*networkConfiguration, error) {
+	config := &networkConfiguration{}
+	if genData, ok := option[netlabel.GenericData]; ok && genData != nil {
+		switch opt := genData.(type) {
+		case map[string]interface{}:
```

Comment:
We have to support labels shortly and hence we should also retain map[string]interface{} for that case.